index.html

```diff
@@ -4,7 +4,54 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Table</title>
   </head>
-  <body></body>
+  <body>
+    <table width="100%" border="10" cellpadding="8" bgcolor="white">
+      <thead>
+        <tr bgcolor="blue" >
+            <th><font color="white">№</font></th>
+            <th><font color="white">Name</font></th>
+            <th><font color="white">Repo</font></th>
+            <th><font color="white">Preview</font></th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr bgcolor="blue" >
+            <td><font color="white">1</font></td>
+            <td><font color="white">CV</font></td>
+            <td><a href="https://github.com/shtefannnn/cv" style="color: #f7f7f5;" target="_blank">not interested</a></td>
+            </td>
```

Comment:
```suggestion
            <td><a href="https://github.com/shtefannnn/cv" style="color: #f7f7f5;" target="_blank">not interested</a></td>
```
Двічі закривається один і той же тег `</td>`